products/sle/main.pm

```diff
@@ -863,8 +863,10 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/login_console";
         loadtest "virt_autotest/install_package";
         loadtest "virt_autotest/update_package";
-        loadtest "virt_autotest/reset_partition";
-        loadtest "virt_autotest/reboot_and_wait_up_normal" if (!get_var('AUTOYAST') && get_var('REPO_0_TO_INSTALL'));
+        # Skip reset_partition for s390x due to there just be 42Gib disk space for each s390x LPAR
+        loadtest "virt_autotest/reset_partition" if (!is_s390x);
+        # Skip reboot_and_wait_up_normal for s390x due to new changes from svirt backend for power_action_utils::power_action (see poo#151786)
+        loadtest "virt_autotest/reboot_and_wait_up_normal" if (!get_var('AUTOYAST') && get_var('REPO_0_TO_INSTALL') && (!is_s390x));
```

Comment:
Good question, generally, we'd better to reboot system after we made new changes in update_package test module.  However, found that new test failure during call reboot_and_wait_up_normal test module due to there are some changes for svirt backend to call power_action_utils.pm, create [poo#151786](https://progress.opensuse.org/issues/151786) - [virtualization][s390x LPAR] new test failuer about reboot_and_wait_up_normal from osd s390x test run

Reply:
Ok, then let's fix it in [poo#151786](https://progress.opensuse.org/issues/151786). I bump the ticket's priority and make it a subtask of [poo#135425](https://progress.opensuse.org/issues/135425). Otherwise we won't be testing the correct host system.

Reply:
Super, gotcha.